core/src/main/java/com/graphhopper/reader/dem/SRTMGL1Provider.java

```diff
@@ -0,0 +1,155 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for 
+ *  additional information regarding copyright ownership.
+ * 
+ *  GraphHopper GmbH licenses this file to you under the Apache License, 
+ *  Version 2.0 (the "License"); you may not use this file except in 
+ *  compliance with the License. You may obtain a copy of the License at
+ * 
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.reader.dem;
+
+import com.graphhopper.util.Helper;
+
+import java.io.*;
+
+/**
+ * SRTMGL1 contains elevation data for most of the world with 1 arc second (~30m) accuracy.
+ * We use the mirror of OpenTopography, as the official SRTMGL1 download requires authentication.
+ * http://opentopo.sdsc.edu/raster?opentopoID=OTSRTM.082015.4326.1
+ * <p>
+ * TODO: In addition, NSF requires oral acknowledgment during all news media interviews.
+ * TODO: This is weird, we have to check if really want to use OpenTopography then, see: http://www.opentopography.org/citations
+ * <p>
```

Comment:
I find this very weird. Maybe we find a different source for the SRTM data.

> In addition, NSF requires oral acknowledgment during all news media interviews

These seems a bit much for just providing a download server :).

Reply:
Indeed a bit strange.

Other question: how much larger is the full world wide data on disc. (CGIAR is 70GB)

Reply:
I build a GeoTIFF earlier this year for fun (from the raw tiles, no hole fixing etc!): http://hannes.enjoys.it/blog/2017/01/one-srtmgl1-geotiff-to-rule-them-all/

The result was ~90GB without overviews, ~120GB with. I used `-co PREDICTOR 2 -co COMPRESS=DEFLATE -co TILED=YES` which from my understanding was ideal.
https://www.datenatlas.de/geodata/public/srtmgl1/

Reply:
Thanks for the details @kannes. I think the newer SRTMGL1 are already hole/void fixed with ASTER GDEM data? 

See - https://lpdaac.usgs.gov/dataset_discovery/measures/measures_products_table/srtmgl1_v003: 
> Voids in the Version 3.0 products have been filled with ASTER Global Digital Elevation Model (GDEM) Version 2.0, the Global Multi-resolution Terrain Elevation Data 2010 (GMTED2010), and the National Elevation Dataset (NED)

Reply:
What do you think about a new section below the software for data licenses [here](https://github.com/graphhopper/graphhopper/blob/master/NOTICE.md) (and explicitly mention what is used by default)?

I would add stuff we have [here](https://graphhopper.com/api/1/docs/attribution/#data) and also gtfs.

Maybe via a table

|source | license | used as default | included in repo |
|---------|-----------|---------|------|
|OSM for road network data | [ODBL](https://www.openstreetmap.org/copyright) | yes | yes
| GTFS NRW | TODO link | no | yes
| SRTM elevation | TODO link | no | yes
| CGIAR elevation | TODO link | no | yes
| SRTMGL1 elevation | TODO link | no | no
|OpenTopography mirror for SRTMGL1 | TODO  link | no | no